packages/api/src/graphql/collection.ts

```diff
@@ -35,17 +36,18 @@ const COLLECTION_EXIST = gql`
 
 const COLLECTION_LIST = gql`
   query CollectionList($databaseName: String!) {
-    collectionList(databaseName: $databaseName)
+    collectionList(databaseName: $databaseName) {
+      name
+      indexes
```

Comment:
Sorry for not being clear, please add more fields if there are any additional fields. You can look at the design for more details. Thanks.